backend/resolvers/types/entryType.js

```diff
@@ -1,19 +1,33 @@
 import { IMAGE_ENTRY, VIDEO_ENTRY, OTHER_ENTRY } from '../../constants'
 
+// For Convenience: the 'base class' of functions for Photo / Video / OtherMedia
+// entry type fields, which they share.
+export const EntryBase = {
+  group (entry) {
+    return entry.getGroup()
+  },
+  student (entry) {
+    return entry.getStudent()
+  },
+  score (entry) {
+    return entry.getScore()
+  }
+}
+
 export default {
   Entry: {
-    group (entry) {
-      return entry.getGroup()
-    },
-    score(entry) {
-      return entry.getScore()
-    },
     __resolveType (data, context, info) {
+      // Identifies for GraphQL which concrete instance of Entry this object is.
+      // Here we also prefetch the associated photo/video/other object so that
+      // their specific type resolvers can pull out those properties
```

Comment:
hmmmmmm can remove the last 2 lines of this comment since we should also remove a bit of what it describes below